inst/WORDLIST

```diff
@@ -0,0 +1,66 @@
+AR
+CMD
+Chracter
```

Comment:
Reviewer: Please delete every word that is invalid.

After that we need to (1) run `spelling::spell_check_package()` to find where the incorrect words are, (2) fix them manually, and (3) run `spelling::update_word_list()`. If you want you can do this and push to either this PR or a separate PR. And if you want I can do it.

Reply:
cool, will do from start to finish

Reply:
done